positron/branding/unofficial/locales/en-US/brand.dtd

```diff
@@ -0,0 +1,8 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<!ENTITY  brandShortName  "Positron">
+<!ENTITY  brandFullName   "Mozilla Positron">
+<!ENTITY  vendorShortName "Mozilla">
+<!ENTITY  logoTrademark   "">
```

Comment:
It isn't clear where this'll be used, although it shouldn't cause any problem to include it.